<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="16" total="17" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-18T09:10:13 IST" name="Default suite" finished-at="2021-08-18T09:10:17 IST" duration-ms="4163">
    <groups>
    </groups>
    <test started-at="2021-08-18T09:10:13 IST" name="Default test" finished-at="2021-08-18T09:10:17 IST" duration-ms="4163">
      <class name="Repeatation.R1">
        <test-method signature="seleniumAlpha()[pri:0, instance:Repeatation.R1@66f57048]" started-at="2021-08-18T09:10:13 IST" name="seleniumAlpha" finished-at="2021-08-18T09:10:17 IST" duration-ms="4116" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Locator must be serializable to JSON using a `toJson` method. By.tagName: label]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Locator must be serializable to JSON using a `toJson` method. By.tagName: label
at org.openqa.selenium.support.locators.RelativeLocator.assertLocatorCanBeSerialized(RelativeLocator.java:263)
at org.openqa.selenium.support.locators.RelativeLocator.access$100(RelativeLocator.java:43)
at org.openqa.selenium.support.locators.RelativeLocator$RelativeBy.<init>(RelativeLocator.java:78)
at org.openqa.selenium.support.locators.RelativeLocator$RelativeBy.<init>(RelativeLocator.java:73)
at org.openqa.selenium.support.locators.RelativeLocator$RelativeBy.<init>(RelativeLocator.java:68)
at org.openqa.selenium.support.locators.RelativeLocator.withTagName(RelativeLocator.java:65)
at Repeatation.R1.seleniumAlpha(R1.java:928)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seleniumAlpha -->
      </class> <!-- Repeatation.R1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
